src/main/java/fr/ens/biologie/genomique/eoulsan/modules/diffana/DESeq2Checker.java

```diff
@@ -35,12 +36,48 @@ static boolean checkExperimentDesign(Experiment experiment) throws EoulsanExcept
      * @param throwsException if true throw an exception
      * @throws EoulsanException if the experiment design is not correct
      */
+
     static boolean checkExperimentDesign(Experiment experiment, boolean throwsException) throws EoulsanException {
 
         requireNonNull(experiment, "Experiment argument cannot be null");
         final ExperimentMetadata emd = experiment.getMetadata();
-        // Check the comparison string, if exists it is a complex design
+        // Get the name of all keys
+        List<String> metakeys = DesignUtils.getExperimentSampleAllMetadataKeys(experiment);
+        List<String> keys = getAllSamplesMetadataKeys(experiment.getDesign());
+        List<String> allkeys= new ArrayList<>(metakeys);
+        allkeys.addAll(keys);
+
+        /* Check if there is an empty cell in the experiment */
+        // For every key, check if there is no empty data
+        for (String key : allkeys) {
+            for (ExperimentSample es : experiment.getExperimentSamples()) {
+                String value = DesignUtils.getMetadata(es, key);
+                if(value.equals(null) || value.equals("")){
```

Comment:
You can use the Guava's Strings.nullOrEmpty() method:
`if (Strings.nullOrEmpty(value))` 
and after static import:
``if (nullOrEmpty(value))`